harness/features/variable.cloud.test.ts

```diff
@@ -0,0 +1,260 @@
+import { 
+    getConnectionStringForProxy, 
+    forEachSDK, 
+    describeIf, 
+    createClient, 
+    createUser,
+    wait,
+    callVariable, 
+    forEachVariableType,
+    variablesForTypes
+} from '../helpers'
+import { Capabilities, SDKCapabilities } from '../types'
+import { v4 as uuidv4 } from 'uuid'
+import { getServerScope } from '../mockServer'
+import nock from 'nock'
+
+jest.setTimeout(1000000)
```

Comment:
isn't this too high?

Reply:
yep, i think elliot changed it for testing and i forgot to change it back